symfony/plugins/orangehrmAdminPlugin/Api/OrganizationAPI.php

```diff
@@ -0,0 +1,276 @@
+    public const PARAMETER_NOTE = 'note';
+
+    /**
+     * @var null|OrganizationService
+     */
+    protected ?OrganizationService $organizationService = null;
+
+    /**
+     * @var EmployeeService|null
+     */
+    protected ?EmployeeService $employeeService = null;
+
+    /**
+     * @return OrganizationService
+     */
+    public function getOrganizationService(): OrganizationService
+    {
+        if (is_null($this->organizationService)) {
+            $this->organizationService = new OrganizationService();
+        }
+        return $this->organizationService;
+    }
+
+    /**
+     * @param OrganizationService $organizationService
+     */
+    public function setOrganizationService(OrganizationService $organizationService): void
+    {
+        $this->organizationService = $organizationService;
+    }
+
+    /**
+     * @return EmployeeService|null
+     */
+    public function getEmployeeService(): ?EmployeeService
+    {
+        if (is_null($this->employeeService)) {
+            $this->employeeService = new EmployeeService();
+        }
+        return $this->employeeService;
+    }
+
+    /**
+     * @param EmployeeService|null $employeeService
+     */
+    public function setEmployeeService(?EmployeeService $employeeService): void
+    {
+        $this->employeeService = $employeeService;
+    }
+
+    /**
+     * @return EndpointGetOneResult
+     * @throws RecordNotFoundException
+     * @throws Exception
+     */
+    public function getOne(): EndpointGetOneResult
+    {
+        // TODO:: Check data group permission
+        $orgInfo = $this->getOrganizationService()->getOrganizationGeneralInformation();
+        if (!$orgInfo instanceof Organization) {
+            throw new RecordNotFoundException();
+        }
+
+        return new EndpointGetOneResult(OrganizationModel::class, $orgInfo);
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function getValidationRuleForGetOne(): ParamRuleCollection
+    {
+        return new ParamRuleCollection(
+            new ParamRule(
+                CommonParams::PARAMETER_ID,
+                new Rule(Rules::POSITIVE)
+            ),
+        );
+    }
+
+    /**
+     * @return EndpointGetAllResult
+     * @throws Exception
+     */
+    public function getAll(): EndpointGetAllResult
+    {
+        throw new NotImplementedException();
+    }
+
+    /**
+     * @inheritDoc
+     * @throws NotImplementedException
+     */
+    public function getValidationRuleForGetAll(): ParamRuleCollection
+    {
+        throw new NotImplementedException();
+    }
+
+    /**
+     * @inheritDoc
+     * @throws Exception
+     */
+    public function create(): EndpointCreateResult
+    {
+        // TODO:: Check data group permission
+        $orgInfo = $this->saveOrganizationInfo();
+
+        return new EndpointCreateResult(OrganizationModel::class, $orgInfo);
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function getValidationRuleForCreate(): ParamRuleCollection
```

Comment:
`throw new NotImplementedException();`, move these rules to update.
refer https://github.com/orangehrm/orangehrm/blob/5.0/symfony/plugins/orangehrmPimPlugin/Api/EmployeeAPI.php#L212 
to define validation rules